README.md

```diff
@@ -2,8 +2,11 @@
 ###Intro to the Back End
  **Required**
  
-What is the difference between front- and back-end development?
-Why do you need to install Ruby but not HTML/CSS?
+**What is the difference between front- and back-end development?**  
+**Frontend** is the part of the web that you can see and interact with.  
+**backend** usually consists of three parts: a server, an application, and a database  
+**Why do you need to install Ruby but not HTML/CSS?**    
+HTML/css are what you see on every site whereas ruby is used on the backend.
```

Comment:
HTML/CSS are interpreted by a browser.  Ruby needs to be interpreted to into machine code which has to be installed